frontend-react/src/pages/daily/Table/ReportsTable.tsx

```diff
@@ -77,36 +76,11 @@ function ReportsTable() {
     const { loadingServices, services, activeService, setActiveService } =
         useReceiverFeeds();
     // TODO: Doesn't update parameters because of the config memo dependency array
-    const {
-        data: deliveries,
-        loading,
-        error,
-        trigger: getReportsList,
-    } = useReportsList(activeMembership?.parsedName, activeService?.name);
-    const filterManager = useFilterManager(filterManagerDefaults);
-
-    useEffect(
-        () => {
-            // IF parsedName and activeService.name are FOR SURE valid values
-            // AND we don't have any deliveries yet (i.e. first fetch *has not* triggered)
-            if (
-                deliveries === undefined &&
-                activeMembership?.parsedName !== undefined &&
-                activeService?.name !== undefined
-            ) {
-                // Trigger useReportsList()
-                getReportsList();
-            }
-        },
-        // Ignoring getReportsList as dep
-        [activeService, deliveries, activeMembership?.parsedName] //eslint-disable-line react-hooks/exhaustive-deps
+    const { serviceReportsList } = useOrgDeliveries(
```

Comment:
This PR _only_ swaps out the use of the deliveries endpoint. We do still rely on old fetch methods for `useReceiverFeeds` as it uses `useResource(OrganizationResource)`.

Reply:
Never let perfect get in the way of better.